apps/api/src/report/report.controller.ts

```diff
@@ -55,12 +59,8 @@ export class ReportController {
   @ApiOperation({ summary: 'Report 4: licensing stage applicants' })
   @Get('/applicant/licensing-stage')
   @AllowAccess(Access.REPORTING)
-  async getLicensingStageApplicants(
-    @Query('from') from: string,
-    @Query('to') to: string,
-  ): Promise<object[]> {
-    const statuses = await this.reportService.getStatusMap();
-    return this.reportService.getLicensingStageApplicants(statuses, from, to);
+  async getLicensingStageApplicants(@Query('period') period: number): Promise<object[]> {
```

Comment:
what do you mean?  I figured the data would all be outdated regardless, since the sync only runs at certain times, so if the caching happens shortly after the sync, the data should be up to date, no?  that's my theory at least

Reply:
What if we want to see the up-to-date report instead of the cached one just after updating new data?
I am sure Akhil wants to do that.

Reply:
ahhh, right, forgot about postman testing!